controller/instance_manager_controller.go

```diff
@@ -190,6 +192,12 @@ func NewInstanceManagerController(
 	}
 	imc.cacheSyncs = append(imc.cacheSyncs, ds.SettingInformer.HasSynced)
 
+	maxPodRecreateBackoff, err := ds.GetSettingAsInt(types.SettingNameMaxPodRecreateBackoff)
+	if err != nil {
+		return nil, err
+	}
+	imc.exponentialBackoff = NewExponentialBackoff(maxPodRecreateBackoff)
```

Comment:
The `maxPodRecreateBackoff` in a controller's `exponentialBackoff` is configured when the controller is created. If the user updates this setting later, the `maxPodRecreateBackoff` in the `exponentialBackoff` will not be updated accordingly. The only method to apply the new setting value is deleting and recreating longhorn-manager pods . Can we improve this?

Reply:
Yes, good point. I'll work on it.